### PR TITLE
dismiss showing save modal by passing option to file of editor view

### LINKED ISSUE #8529: When you click cmd-S, stack editor tries to save file like a filetree.
image

### PR DESCRIPTION
**Pull request by hakankaradis**
fixes #8529

### CODE REVIEW

File: client/stacks/lib/views/stacks/editors/basestackeditorview.coffee [hunk 1]
```diff
@@ -31,6 +31,7 @@ module.exports = class BaseStackEditorView extends IDEEditorPane
     options.contentType = contentType
     options.file        = FSHelper.createFileInstance
       path: "localfile:/stack-#{Date.now()}.#{contentType}"
+      dismissSaveDialogModal: yes
```
Comment (gokmen) [inline client/stacks/lib/views/stacks/editors/basestackeditorview.coffee]: this option is not related with the `File` instance here, instead you can `off` `'ace.requests.save'` event on ace instance.

Comment (hakankaradis) [inline client/stacks/lib/views/stacks/editors/basestackeditorview.coffee]: 👍
